packages/ui/primitives/document-flow/add-fields.tsx

```diff
@@ -316,7 +316,7 @@ export const AddFieldsFormPartial = ({
           {selectedField && (
             <Card
               className={cn(
-                'bg-background pointer-events-none fixed z-50 cursor-pointer transition-opacity',
+                'bg-documenso pointer-events-none fixed z-50 cursor-pointer border-2 bg-opacity-20 backdrop-blur-[1.2px]',
```

Comment:
I find it quite difficult to see the fields properly with these values. What do you think?

![CleanShot 2024-02-02 at 14 46 39](https://github.com/documenso/documenso/assets/25515812/a3686d1e-883d-46b0-a070-e63dc3b731a6)

I played a bit with the values, and managed to get to this version. What do you think?

```
bg-documenso h-full w-full bg-opacity-80 backdrop-blur-[1px]
```

![CleanShot 2024-02-02 at 14 51 38](https://github.com/documenso/documenso/assets/25515812/6120a7d0-297d-4ac6-a5c4-650b2912dc4a)

Or, here's another example (maybe closer to the design spec?).

```
bg-documenso/60 h-full w-full bg-opacity-80 backdrop-blur-[1px]
```

![CleanShot 2024-02-02 at 14 55 40](https://github.com/documenso/documenso/assets/25515812/f0019dcf-0b2d-4b46-96c7-f37a1e63a971)

Anyway, just my thoughts. Feel free to discard them. 😆

Reply:
I agree with Catalin, it's hard to see

IMO transparent when dragging and not transparent when dropped would be optimal

Reply:
Idk why but the design one seems to be way better. I feel the background values would be good in the ranges of 30-40 😶

Reply:
Yup, the design one looks the best. Couldn't replicate it 1-1 though.

At the moment, it's a bit difficult to see the field text when it's placed on top of the component. Any ideas on how we can overcome this?

Other than that, it looks good to me.

<img width="263" alt="CleanShot 2024-02-19 at 09 36 43@2x" src="https://github.com/documenso/documenso/assets/25515812/fa20123c-1b4f-44ac-bb8b-23b647555c49">